src/main/java/in/hp/java/userservice/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package in.hp.java.userservice.service;
+
+import in.hp.java.userservice.model.User;
+import in.hp.java.userservice.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    public void addUser(User user) {
+        userRepository.save(user);
+    }
+
+    public List<User> getUsers() {
+        return userRepository.findAll();
+    }
+
+    public User getUser(Long id) {
```

Comment:
What if we dont have that User id? please handle Exception using Controller Advise

Reply:
Added Exception handlers